cloud-core/src/main/java/cloud/commandframework/CommandManager.java

```diff
@@ -1417,7 +1417,16 @@ public enum ManagerSettings {
          * @since 1.2.0
          */
         @API(status = API.Status.STABLE, since = "1.2.0")
-        OVERRIDE_EXISTING_COMMANDS
+        OVERRIDE_EXISTING_COMMANDS,
+
+        /**
+         * When enabled, optional flag arguments are appended between each command node.
+         * It can have some conflicts when integrating with other command systems like brigadier.
+         *
+         * @since 1.8.0
+         */
+        @API(status = API.Status.STABLE, since = "1.8.0")
```

Comment:
probably go with experimental here as well if we do on the other related new methods